### ISSUE TITLE
Feature : Add some more options to pg:pull through pg_dump

### ISSUE THREAD

**Issue by BenTalagan** [heroku/heroku-pg#159]
Hello,

A very useful option has been lost when removing taps from the cli : the ability to exclude some table datas based on a table name. I wish it were back. It seems to me possible (not totally sure), looking at the code from pull.js where the dump command is defined as :

```
let dump = `env${password(source.password)} PGSSLMODE=prefer pg_dump --verbose -F c -Z 0 ${connstring(source, true)}`
```

Thus a few more options could be passed to add provide more pg_dump features, especially the **--exclude-table-data=MYTABLE** option.

What about re-adding this ? :) Thanks!

**Comment by msakrejda** [comment]
That seems like a worthwhile option to support, but realistically, we have a lot of things in our backlog and we likely won't get around to this anytime soon. If someone wants to contribute a patch to support this, though, I'm happy to review.

**Comment by BenTalagan** [comment]
Here you go, then : PR #160 😃

**Comment by msakrejda** [comment]
Shipped in #160--thanks again!
